modules/profile/manifests/python.pp

```diff
@@ -21,6 +21,22 @@
   Exec['update-pip'] -> Python::Pip<||>
   Class['::st2::profile::repos'] -> Class['::st2::profile::python']
 
+  if $osfamily == 'RedHat' {
+    file { '/etc/facter/facts.d/six_upgrade_20151012.txt':
+      ensure  => file,
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0644',
+      content => 'six_upgrade_20151012=true',
+      notify  => Package['python-six-1.9.0-1.el7.noarch.rpm'],
+    }
+    package {'python-six-1.9.0-1.el7.noarch.rpm':
+      ensure   => 'present',
+      provider => 'rpm',
+      source   => 'http://cbs.centos.org/kojifiles/packages/python-six/1.9.0/1.el7/noarch/python-six-1.9.0-1.el7.noarch.rpm'
```

Comment:
Let me check what this does for RHEL 6 before we merge. This might need to be fenced to RHEL 7 only for now.